app/controlplane/pkg/biz/group.go

```diff
@@ -824,3 +834,11 @@ func (uc *GroupUseCase) ValidateGroupIdentifier(ctx context.Context, orgID uuid.
 
 	return groups[0].ID, nil
 }
+
+// isReservedGroupName checks if the provided group name is a reserved name.
+// Reserved names are used by the system for default roles and cannot be used for custom groups.
+// Any name starting with "Org-" (case insensitive) is considered reserved.
+func isReservedGroupName(name string) bool {
+	// Check if the name starts with "org-" (case insensitive)
+	return strings.HasPrefix(strings.ToLower(name), "org-")
```

Comment:
I would move this to a configuration setting and make it optional. What do you think?